src/main/webapp/app/modules/conflicts/shared/shared-record-view.reducer.ts

```diff
@@ -75,6 +91,16 @@ export default (state: SharedRecordViewState = initialState, action): SharedReco
         dismissedMatches,
         hiddenMatches
       };
+    case ACTION_TYPES.OPEN_SERVICE:
+      return {
+        ...state,
+        openService: action.payload
+      };
+    case ACTION_TYPES.OPEN_LOCATION:
+      return {
+        ...state,
+        openLocation: action.payload
```

Comment:
I would rename this to openedLocation / selectedLocation (same for method names)